frontend/static/js/pages/play.js

```diff
@@ -188,7 +210,7 @@ let app = new Vue({
                 this.endTime = Date.now();
             }
 
-            submitRun(PROMPT_ID, LOBBY_ID, this.runId, this.startTime, this.endTime, this.finished, this.path);
+            submitRun(PROMPT_ID, LOBBY_ID, this.runId, this.startTime, this.endTime, this.finished, this.path, this.penaltyTime);
```

Comment:
This shouldn't work... I think. Since our path is a JSON object, we can just serialize penaltyTime as part of that without having to modify submitRun. Writing example down below